src/repositories/implementations/DynamoDB/GetVolunteersDynamoRepository.ts

```diff
@@ -51,6 +56,11 @@ export class GetVolunteersDynamoRepository
 
         const lastEvaluatedParsed = LastEvaluatedKey as TlastEvaluatedParsed;
 
-        return [lastEvaluatedParsed, volunteersList];
+        const content: ResponseGetVolunteers = {
+            id: lastEvaluatedParsed,
+            volunteers: volunteersList,
```

Comment:
Initialize a new entity for all the volunteers.
for example `volunteersList.map(volunteer => new Volunteer(volunteer))`